server/rest/dataset.py

```diff
@@ -400,3 +403,84 @@ def createGeojson(self, item, params):
         .param('endTime', 'latest time to include result', required=False)
         .errorResponse('ID was invalid.')
         .errorResponse('Write permission denied on the Item.', 403))
+
+    @access.public
+    @autoDescribeRoute(
+        Description('Download a dataset.')
+        .modelParam('id', model='item', level=AccessType.READ)
+        .errorResponse('ID was invalid.')
+        .errorResponse('Read access was denied on the parent folder.', 403))
+    def download(self, item, params):
+        minervaMeta = item['meta']['minerva']
```

Comment:
I am wondering if in the future, we can use some sort of registration to register code to download data depending on the data format. The current solution is fine for now.